src/drivers/distance_sensor/lightware_laser_i2c/lightware_laser_i2c.cpp

```diff
@@ -259,6 +259,9 @@ int LightwareLaser::enableI2CBinaryProtocol()
 		return ret;
 	}
 
+	// Wait a bit, otherwise the next transfer occasionally returns ret == value[0] == value[1] == 0
```

Comment:
or check for that condition before sleeping and retrying?

Reply:
Alright